src/components/Button.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Hi, I would advise you not to disable the linter, Can you fix the mistake you are getting manually?
- this is because a [linter is a tool](https://github.com/microverseinc/linters-config) that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs